### DISCUSSION TITLE
customized Icon Sort

### DISCUSSION THREAD

**Discussion by ashkankiani** [komarovalexander/ka-table#234] [Q&A] [answered]
How to change the sort icon. customized?

**Comment by komarovalexander** [comment, marked as answer]
Hi @ashkankiani 
good question!
`childComponents.sortIcon` was added in `v7.6.0`
details here:  https://github.com/komarovalexander/ka-table/releases/tag/7.6.0

**Reply by ashkankiani** [reply]
@komarovalexander Thank You!
It was the right way to do it.

**Reply by ashkankiani** [reply]
hint : At first, I tried a lot and it didn't work, but I noticed that version 7.6.0 was released a few hours ago, so make sure to update it.

**Reply by komarovalexander** [reply]
Exactly. This sentence is about that “was added in v7.6.0” 🙂
